### PR TITLE
[Doc Issue][Documentation section needs to be updated for "Toggle incoming audio"][2560995]

### PR DESCRIPTION
**Pull request by v-silakshmi**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/162934bf5efabd77b561229876ae617098b28ca3/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 92 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/efdc9d15-7579-41be-ba06-a5ee1d2ea2f1) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/352de2981b5072d8055ba287ffc52e2327d91d99/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 92 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/16064316-90f1-4820-8714-edbc8b9fd281) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9ddbe4e69ee5f6e13c42bf1a2519c394171ec804/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 92 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c7fffd31-d3a6-474d-9c34-09cb493aa332) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5192b5fdb0437708e047de5b5c69a4df439b1451/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 92 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/06b5384f-8fc7-44f3-8578-62fc0bd75702) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-habibaumme — DISMISSED** [review]
Approved

File: msteams-platform/apps-in-teams-meetings/API-references.md [hunk 1]
```diff
@@ -995,6 +995,41 @@ function toggleIncomingClientAudio(callback: (error: SdkError | null, result: bo
...
+    }
+
+microsoftTeams.meeting.getIncomingClientAudioState(this.callback)
+
+```
+
+> [!NOTE]
+> RSC has been enabled after 1.11, this feature won't work before 1.12.
+
+### Manifest
```
Comment (Wajeed-msft) [inline msteams-platform/apps-in-teams-meetings/API-references.md:1017]: Not sure what we mean by Manifest in this context. Could you please check?
image

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/182d45adac336b31f57d5afa03f6a28b266d0775/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 92 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b4a38b19-ae6b-45d3-9aa9-4a0d0ad36c90) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/apps-in-teams-meetings/API-references.md (hunk 1 above)
Comment (ChetanSharma-msft) [reply msteams-platform/apps-in-teams-meetings/API-references.md]: Hello @Wajeed-msft - We wanted to add the section like we have at other places like below:
image


@v-silakshmi - Can you please update it accordingly.

File: msteams-platform/apps-in-teams-meetings/API-references.md [hunk 2]
```diff
@@ -982,7 +982,7 @@ The following table includes the query parameter:
 |---|---|----|---|
 |**callback**| String | Yes | Callback contains two parameters `error` and `result`. The *error* can either contain an error type `SdkError` or `null` when the toggle is successful. The *result* can either contain true or false value, when the toggle is successful or null when the toggle fails. The incoming audio is muted if the result is true and unmuted if the result is false. |
 
-### Example
+### Syntax
```
[the remark this answers is not in the document]
Comment (v-silakshmi) [reply msteams-platform/apps-in-teams-meetings/API-references.md]: Syntax section is removed as suggested.

File: msteams-platform/apps-in-teams-meetings/API-references.md (hunk 1 above)
Comment (ChetanSharma-msft) [reply msteams-platform/apps-in-teams-meetings/API-references.md]: image

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2977e6d5ab64e9335ce23082f311a4106ba910af/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 92 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/afca7b72-bace-4bfa-b94a-c336e384ffb4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ea56bece4dadb58ff274434b2ceb5f393902c630/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 92 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/19e1e8fe-b29e-482b-9299-98729033b933) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/apps-in-teams-meetings/API-references.md [hunk 3]
```diff
@@ -980,17 +980,56 @@ The following table includes the query parameter:
...
+callback = (error, result) => {
+        if (error) {
+            // Handle error code
+        }
+        else {
+            // Handle success code
+        }
+    }
+
+microsoftTeams.meeting.getIncomingClientAudioState(this.callback)
```
Comment (ChetanSharma-msft) [inline msteams-platform/apps-in-teams-meetings/API-references.md:997]: Can you please replace this method getIncomingClientAudioState() => with toggleIncomingClientAudio()

microsoftTeams.meeting.toggleIncomingClientAudio(this.callback)

Comment (v-silakshmi) [reply msteams-platform/apps-in-teams-meetings/API-references.md]: Replaced the code as suggested.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1d9a417931af4cc3d81f1830bfb7dcc20314b477/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 92 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e5f50f2a-1203-4180-a918-5bbed8012034) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/apps-in-teams-meetings/API-references.md [hunk 4]
```diff
@@ -980,17 +980,56 @@ The following table includes the query parameter:
...
+```JSON
+
+"authorization": {
+    "permissions": {
+      "resourceSpecific": [
+        {
+          "name": "OnlineMeetingParticipant.ToggleIncomingAudio.Chat",
+          "type": "Delegated"
+        }
+      ]
```
Comment (ChetanSharma-msft) [inline msteams-platform/apps-in-teams-meetings/API-references.md:1031]: Added few more sections to update, so please check those as well and update in this PR.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my comments.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f49bd14bbee9d49e375cddc5f026cbbb365ebc36/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 92 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/828493d1-e937-4321-9745-c0cbfdaaf3b3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/apps-in-teams-meetings/API-references.md (hunk 4 above)
Comment (v-silakshmi) [reply msteams-platform/apps-in-teams-meetings/API-references.md:1031]: I have incorporated the suggested comments.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2ca5e7555b34120ed8f254b1cc13e3d0ed840038/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 92 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3178d5c4-edb3-415b-b4cd-f52a2bd24648) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/36fa6bc023d121d15d0e281762c0aa2da51c7530/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 92 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/351520d6-dce5-4217-b2f4-c3d97cc5511f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fd15bc606e959a5953d8444dfeab9eee1c7dfeb6/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 92 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a7f8955f-5b1e-4bc2-830f-5b506484e2d6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fdadf871e50d065c3c7beab1233b7e6128a6b507/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 92 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9b650084-8d1e-4d59-95ac-b9cc3b0e4b64) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by ChetanSharma-msft — DISMISSED** [review]
Changes looks fine, approving it.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b727bc46bc1a60c15e723a83d275eea0ccffffe7/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 92 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4992a6e9-9b44-4eab-a78b-d5784fc272a2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/18dec667465e3a04fdf3e71652f7a667e0ddfa4b/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 92 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0b56ee3d-b6b6-48f6-a1d7-6bf9ac18ea94) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6d382ce18b174f828e7fd01e21993bc68bab15ab/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 92 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e73de1b4-a11e-482b-9c7e-ee1244c321c4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/17be39fdd1326cc5d839160d53642f57b89f95e6/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 92 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/837478e4-970f-4c61-b4bf-64061fee48bb) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Wajeed-msft — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**